### PR TITLE
chore(tests) Add test harness

### PR DESCRIPTION
**Pull request by wanthony**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,6 +24,11 @@
     "gulp-typescript": "^2.5.0",
     "gulp-util": "^3.0.4",
     "inquirer": "^0.8.2",
+    "jasmine-core": "^2.2.0",
+    "karma": "^0.12.31",
+    "karma-jasmine": "^0.3.5",
+    "karma-phantomjs-launcher": "~0.1",
```
Comment (goldcaddy77) [inline package.json]: Can we keep the same format for version # here?

File: package.json [hunk 2]
```diff
@@ -24,6 +24,11 @@
...
+    "karma-typescript-preprocessor": "git://github.com/wanthony/karma-typescript-preprocessor",
```
Comment (goldcaddy77) [inline package.json]: We should probably get off of hard-coding your pre-processor here at some point.  3 options:

* Possibly create your own?
* Get your PR into the main `karma-typescript-preprocessor` project?
* Is there another preprocessor?

Thoughts?

**Comment by goldcaddy77** [comment]
Looks good - just had a few questions.  For the `karma-typescript-preprocessor` question, we can start with pointing at your fork, but we should figure out a plan there.

**Comment by wanthony** [comment]
@goldcaddy77 updated.

**Pull request merged**